framework/src/controller/application.js

```diff
@@ -8,8 +8,8 @@ const { createLoggerComponent } = require('../components/logger');
 
 const ChainModule = require('../modules/chain');
 
-// Private scope used because private keyword is restricted
-const scope = {
+// Private __private used because private keyword is restricted
+const __private = {
```

Comment:
storage is using `_` convention, so maybe just single underscore is ok?

Reply:
@shuse2 We are using `__` syntax already in the code everywhere. 

https://github.com/LiskHQ/lisk/blob/cb68ebb96aacc3dccd532de6f926a1c39a2b1411/modules/loader.js#L30
https://github.com/LiskHQ/lisk/blob/cb68ebb96aacc3dccd532de6f926a1c39a2b1411/modules/blocks/verify.js#L35

The convention of `_` was used for parameters which are not used in the functions.

Reply:
Let's create a seperate issue to refactor the coding style when we have a documented one. And then change all the code.